modules/manage/pages/schema-reg/schema-reg-api.adoc

```diff
@@ -76,6 +76,49 @@ def pretty(text):
 base_uri = "http://localhost:8081"
 ----
 
+== Manage Schema Registry ACLs
+
+ifndef::env-cloud[]
+[NOTE]
+====
+include::shared:partial$enterprise-license.adoc[]
+====
+endif::[]
+
+You can use ACLs to control access to Schema Registry resources. You can define fine-grained access on a global level, for example, to allow a principal to read all schemas, or on a per-subject basis; for example, to read and write to only the schemas of a specific subject.
+
+See xref:manage:schema-reg/schema-reg-authorization.adoc[] for more details on Schema Registry Authorization.
+
+For example, to xref:api:ROOT:pandaproxy-schema-registry.adoc#post-/security/acls[create ACLs] that allow users with the `admin` role read-only access to all registered schemas, run:
+
+[,bash]
+----
+curl -X POST "http://localhost:8081/security/acls" \
+  -H "Content-Type: application/json" \
+  -d '[
+    {
+      "principal": "RedpandaRole:admin",
+      "resource": "*",
+      "resource_type": "REGISTRY",
+      "pattern_type": "LITERAL",
+      "host": "*",
+      "operation": "DESCRIBE_CONFIGS",
```

Comment:
@Feediver1 just an observation but the resources/operations are all caps here and in the main ACLs docs operations use a different capitalization, so slightly harder to spot.

Reply:
> @Feediver1 just an observation but the resources/operations are all caps here and in the main ACLs docs operations use a different capitalization, so slightly harder to spot.
> 
> 

Technically my fault; was trying to be a bit more appropriate with swagger by using uppercase for enums, but in the end it goes against consistency.

Reply:
@BenPope @mattschumpert this doc follows the provided Swagger definition (reference [here](https://docs.redpanda.com/api/schema-registry-api/#get-/security/acls)) so I'll keep it to all caps for now unless we update the definition later. cc @Feediver1